### DISCUSSION TITLE
swoole vs openswoole

### DISCUSSION THREAD

**Discussion by amir-khoshbakht** [swoole/swoole-src#4678] [Q&A]
what is different between swoole and openswoole ?

**Comment by NathanFreeman** [comment]
Hi.
Openswoole is forked from Swoole.
They have a very similar codebase.
For now, there's little difference between Swoole and Openswoole.

**Comment by rela589n** [comment]
Which one is better to use then?

**Reply by rela589n** [reply]
@amir-khoshbakht , see also https://github.com/openswoole/ext-openswoole/issues/28
